voxygen/src/game.rs

```diff
@@ -241,11 +252,18 @@ impl Game {
             renderer.render_model_object(&model);
         }
 
+        /* // Draw ui
+        let size = self.window.get_size();
+
+        self.ui.set_size(size[0] as u32, size[1] as u32);
```

Comment:
@xMAC94x this needs mutably reference

Reply:
yeah i noticed this afterward, can you change the Ui in line 46 to a Mutex<Ui> to fit in the other structures